composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": ">=5.5.9",
-        "cakephp/cakephp": "3.3.*",
+        "cakephp/cakephp": "~3.3",
```

Comment:
Using `~3.3` would auto update to 3.4 when it's released and based on past experience I am :-1: on auto updating to  next minor version.